FreeSurfer/PostFreesurferLongTPUpdate.sh

```diff
@@ -0,0 +1,38 @@
+#As 12/2023, this script is deprecated and should not be used. 
+
+#!/bin/bash
```

Comment:
This line needs to be the first line in the file in order for it to be fully useful (it tells the OS what interpreter to run it with).